### PR TITLE
Add heirlooms to collections

### PR DESCRIPTION
**Pull request by seirl**

### CODE REVIEW

File: src/components/Nav.svelte [hunk 1]
```diff
@@ -32,7 +32,8 @@
 				{ txt: 'Companions',  link: 'companions' },
 				{ txt: 'Battle Pets', link: 'battlepets' },
 				{ txt: 'Toys',        link: 'toys'       },
-				{ txt: 'Titles',	  link: 'titles'     }
+				{ txt: 'Titles',	  link: 'titles'     },
+				{ txt: 'Heirlooms',	  link: 'heirlooms'  },
```
Comment (maxdekrieger) [inline src/components/Nav.svelte:36]: Heirlooms feels more like a collectable than titles, imo we can put heirlooms above titles in the menu

File: src/api/heirlooms.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    // combine
+    _cache.update(
+        region,
+        realm,
+        character,
+        parseCollectablesObject(db, profile, collected, 'heirlooms', 'heirloom')
+    )
+    return _cache.cache;
+}
```
Comment (maxdekrieger) [inline src/api/heirlooms.js:41]: parallel to this pr but all the JS API files for collectables are the same except for a few keywords, maybe we can generalize this one day.

File: static/data/heirlooms.json [hunk 3]
```diff
@@ -0,0 +1,952 @@
...
+            "ID": 687,
+            "icon": "inv_shoulder_29",
+            "itemId": 122375,
+            "name": "Aged Pauldrons of The Five Thunders"
+          },
+          {
+            "ID": 688,
+            "icon": "inv_jewelry_trinketpvp_01",
+            "itemId": 122371,
+            "name": "Inherited Insignia of the Alliance"
```
Comment (maxdekrieger) [inline static/data/heirlooms.json:640]: (Alliance only)

File: src/api/heirlooms.js (hunk 2 above)
Comment (seirl) [reply src/api/heirlooms.js:41]: Yes, was thinking the same.

File: static/data/heirlooms.json [hunk 4]
```diff
@@ -0,0 +1,952 @@
...
+            "ID": 678,
+            "icon": "inv_hammer_07",
+            "itemId": 122367,
+            "name": "The Blessed Hammer of Grace"
+          },
+          {
+            "ID": 679,
+            "icon": "inv_jewelry_trinketpvp_02",
+            "itemId": 122370,
+            "name": "Inherited Insignia of the Horde"
```
[the remark this answers is not in the document]
Comment (seirl) [reply static/data/heirlooms.json:586]: They do show up for both factions (just like toys), so we don't need to.

**Pull request merged**